packages/experimental-utils/src/ts-eslint/Rule.ts

```diff
@@ -265,6 +269,7 @@ interface RuleListener {
   JSXFragment?: RuleFunction<TSESTree.JSXFragment>;
   JSXIdentifier?: RuleFunction<TSESTree.JSXIdentifier>;
   JSXMemberExpression?: RuleFunction<TSESTree.JSXMemberExpression>;
+  // JSXNamespacedName?: RuleFunction<TSESTree.JSXNamespacedName>; see: https://github.com/Microsoft/TypeScript/issues/7411
```

Comment:
I would just remove this from here and from the `AST_NODE_TYPES` enum.
It's never going to be added, so there's no reason to keep dead code around.